src/main.c

```diff
@@ -528,6 +549,13 @@ int main(int argc, char* argv[]) {
   if (getenv("JQ_COLORS") != NULL && !jq_set_colors(getenv("JQ_COLORS")))
       fprintf(stderr, "Failed to set $JQ_COLORS\n");
 
+  // Randomness
+  if (options & USER_DEFINED_SEED) {
```

Comment:
Why would we want user-defined seeds?

Reply:
The reason why I added in a user defined seed is for reproducability. Often time I need a source of numbers that seem random, but I also want to guarantee the the result is deterministic. That's why I added this option.